src/pytest_cpp/boost.py

```diff
@@ -50,15 +61,11 @@ def read_file(name: str) -> str:
                 return ""
 
         with tempfile.TemporaryDirectory(prefix="pytest-cpp") as temp_dir:
-            log_xml = os.path.join(temp_dir, "log.xml")
-            report_xml = os.path.join(temp_dir, "report.xml")
-            args = list(harness) + [
-                executable,
-                "--output_format=XML",
-                "--log_sink=%s" % log_xml,
-                "--report_sink=%s" % report_xml,
-            ]
+            log_xml = os.path.join(os.path.relpath(tmp_dir), "log.xml")
```

Comment:
Are you making the path relative to `tmp_dir`, and I think you intended for the process to execute in the `temp_dir` (seems there is a typo here), but you never pass `cwd` to `subprocess.Popen`, so this will not work.

Also changing the cwd can be considered a breaking change, now that I think more about it...

Reply:
I think I commited wrong branch (it is about a typo), will check it.
`cwd` doesn't work in case of using Wine because, again, it will be a local unix-path (/home/user/project/bla-bla-bla/tmp/pytestcpp-qergqr12) and wine can work only with windows format (c:\Users\project\bla-bla-bla\tmp\pytestcpp-qergqr12) however if I'm using relative path for a temp folder (like ../../../tmp/pytestcpp-qergqr12) wine can handle it properly.

Reply:
Ahh OK I misunderstood the patch, `relpath` will make a relative path from tmp_dir to cwd already.

We need to account for this though:

> On Windows, ValueError is raised when path and start are on different drives.

In this case, catching the error and falling back to the full path seems reasonable.